.github/workflows/update-snapshot.yml

```diff
@@ -13,14 +13,16 @@ jobs:
     - uses: cachix/install-nix-action@v18
       with:
         nix_path: nixpkgs=channel:nixos-unstable
-    - name: Update snapshot commit
+    - name: Update HEAD snapshot commit
       run: 'nix-shell -p niv --run "niv update emacs-snapshot"'
+    - name: Update release snapshot commit
+      run: 'nix-shell -p niv --run "niv update emacs-release-snapshot"'
```

Comment:
> Does https://github.com/purcell/nix-emacs-ci/pull/135 look sane?

Yes. (As far as I can tell.) But let the nitpicking commence:

Below you use the opposite (and more logical) order. (The release branch is "closer" to a release than the `master` branch.)

Other than that, the only thing I would maybe change are the job names: "Update emacs-29/master snapshot". Not sure about the "... commit" suffix; `niv update <pkg>` doesn't just checkout the commit, it builds the package, right? (I'm slowly getting familiar with guix, but have never used nix.)

Reply:
> Below you use the opposite (and more logical) order. (The release branch is "closer" to a release than the master branch.)

Hehe, fair!

> niv update <pkg> doesn't just checkout the commit, it builds the package, right?

No, `niv` just updates the pinned source. Which would result in a rebuild later.